ibis/backends/tests/test_vectorized_udf.py

```diff
@@ -76,6 +88,8 @@ def demean_struct(v, w):
     output_type=dt.Struct(['mean', 'mean_weight'], [dt.double, dt.double]),
 )
 def mean_struct(v, w):
```

Comment:
are these tested for both inputs?

Reply:
Not sure what you mean by that. Could you clarify?

Reply:
so some udf's only receive Series some can receive ndarray & Series. where is this determined and is this tested (that can get either input)

Reply:
From talking to @icexelloss -  it actually seems to be a property of the window implementation of the backend, not anything to do with the UDF or what the user requests.

Reply:
Yeah this is an existing issue. Window aggregation UDF takes np.ndarray and groupby aggregation takes pd.Series.